### PR TITLE
Uniswap LP token fixes

### PR DESCRIPTION
**Pull request by christianbaroni**
before / after:
Screen Shot 2020-11-17 at 4 52 14 PM

### CODE REVIEW

File: src/components/expanded-state/liquidity-pool/LiquidityPoolExpandedStateHeader.js [hunk 1]
```diff
@@ -45,7 +46,15 @@ function LiquidityPoolExpandedStateHeader({ assets, pricePerShare }) {
   const shadows = useMemo(() => {
     return map(assets, (asset, index) => {
       const coinIconShadow = [
-        [0, 4, 12, asset.shadowColor || colors[index], 0.3],
+        [
+          0,
+          4,
+          12,
+          asset.symbol === 'ETH'
```
Comment (jinchung) [inline src/components/expanded-state/liquidity-pool/LiquidityPoolExpandedStateHeader.js:53]: can we do asset.address === 'eth' so we're relying less on symbols (only the real ETH will have this address)

File: src/components/token-info/TokenInfoBalanceValue.js [hunk 2]
```diff
@@ -4,15 +4,18 @@ import { magicMemo } from '../../utils';
...
   const { address, balance, shadowColor, symbol, value } = asset;
 
   const color = useColorForAsset(asset);
-  const coinIconShadow = useMemo(() => [[0, 3, 9, shadowColor || color, 0.2]], [
-    color,
-    shadowColor,
-  ]);
+  const coinIconShadow = useMemo(
+    () => [
+      [0, 3, 9, symbol === 'ETH' ? colors.dark : shadowColor || color, 0.2],
```
Comment (jinchung) [inline src/components/token-info/TokenInfoBalanceValue.js:15]: same here to use the address === 'eth'

File: src/components/token-info/TokenInfoBalanceValue.js [hunk 3]
```diff
@@ -28,7 +31,9 @@ const TokenInfoBalanceValue = ({ align, asset, ...props }) => {
         size={20}
         symbol={symbol}
       />
-      <TokenInfoValue color={color}>{balance?.display || value}</TokenInfoValue>
+      <TokenInfoValue color={symbol === 'ETH' ? colors.dark : color}>
```
Comment (jinchung) [inline src/components/token-info/TokenInfoBalanceValue.js:34]: and here

**Review by jinchung — APPROVED** [review]

**Review by skylarbarrera — APPROVED** [review]
💯

**Pull request merged**